typescript/lib/parsers/hf.ts

```diff
@@ -240,7 +241,15 @@ export class HuggingFaceTextGenerationParser extends ParameterizedModelParser<Te
     }
 
     if (output.output_type === "execute_result") {
-      if (typeof output.data === "string") {
+      if (output.data?.hasOwnProperty("value")) {
+        const outputData = output.data as OutputData;
+        if (typeof outputData.value === "string") {
+          return outputData.value;
+        }
+        // Sarmad + Ryan, let me know if you prefer I do something else
+        // Was basing this off existing code for function calls in openai.ts
```

Comment:
Lol @rholinshead + @saqadri let me know if you prefer me to return entire JSON vs. a specific field within function call (ex: we can pass in `name`: https://github.com/lastmile-ai/aiconfig/blob/main/typescript/types.ts#L182-L195)

Reply:
I think this is good. Just add a comment so it's clear what the function is trying to do.

Perhaps something like "JSON.stringify to cover the case where output.data.kind is type "tool_calls". Any string values are returned back as-is"